catalog/imaging/imaging-actionprovider-chip/src/main/java/org/codice/alliance/imaging/chip/actionprovider/ImagingChipActionProvider.java

```diff
@@ -151,4 +154,16 @@ private boolean hasValidLocation(String location) {
 
         return hasValidLocation;
     }
+
+    private String getQualifierForRemoteResource(String uriString) throws URISyntaxException {
+        final String QUALIFIER_KEY = "qualifier";
```

Comment:
Should this be a constant? If we keep it this way, it should be camel-cased instead of all-caps.